ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -366,12 +364,21 @@ async function getStatus(node: FullNode): Promise<GetNodeStatusResponse> {
       speed: MathUtils.round(node.workerPool.speed?.rate5s ?? 0, 2),
     },
     accounts: {
-      scanning: accountsScanning,
       enabled: node.config.get('enableWallet'),
       head: {
-        hash: node.wallet.chainProcessor.hash?.toString('hex') ?? '',
-        sequence: node.wallet.chainProcessor.sequence ?? -1,
+        hash: '',
+        sequence: -1,
```

Comment:
Noting that this is a breaking change (though more to what's expected. and it might have been broken already by #4964).

Reply:
Added a note for this.

Reply:
I'd be okay with removing it. I know it's used in the node app but I'm not sure if any external consumers use it. I almost feel like it'd be better to either remove it, or just mark it as deprecated and replace it with `getEarliestHeadHash` (should be pretty much the same thing)